### ISSUE TITLE
Fix sampling to use absolute paths mirroring greedy.

### ISSUE THREAD

**Issue by eparish1** [Pressio/rom-tools-and-workflows#178]
greedy (see [link](https://github.com/Pressio/rom-tools-and-workflows/blob/develop/romtools/workflows/greedy/run_greedy.py) ) looks like this now:
```
def run_greedy(fom_model: QoiModel,
               rom_model_builder: QoiModelWithErrorEstimateBuilder,
               parameter_space: ParameterSpace,
               absolute_greedy_work_directory: str,
               tolerance: float,
               testing_sample_size: int = 10,
               random_seed: int = 1):
```
while [sampling](https://github.com/Pressio/rom-tools-and-workflows/blob/develop/romtools/workflows/sampling/sampling.py) looks like:
```
def run_sampling(model: Model,
                 parameter_space: ParameterSpace,
                 run_directory_prefix: str = "./run_",
                 number_of_samples: int = 10,
                 random_seed: int = 1):
```
we need to fix the second one to do something similar for the paths/prefix (obviously fixing tests too)

**Comment by fnrizzi** [comment]
we are taking this on

**Comment by cwschilly** [comment]
@eparish1 Hi Eric, just wanted to make sure I understand the end goal here.

In `run_sampling`, should the user pass the absolute path to `./` (using the default case as an example), and then we automatically generate `run_{sample_index}` directories within that provided directory?

Or should the user provide the absolute path all the way down to `./run_`, and then we just tag on the sample index to create the directories?

Basically: should the user provide the run directory prefix still or should we assume it's always `run_`?
